apps/web/app/api/user/me/route.ts

```diff
@@ -61,6 +67,10 @@ async function getUser({ userId }: { userId: string }) {
   return {
     ...user,
     members,
+    // Server-side config passed to client
+    supportsOllama,
+    supportsLmStudio,
+    allowUserAiProviderUrl,
```

Comment:
this doesnt have anything to do with the user, so not sure why we need this here?